exercises/matthew_bowles/ex01_basics/inc/main.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
```

Comment:
You don't find `main.h` often. It would be good to put your functions into a separate `.h/.cpp` , eg `WordManipulation.h` and `WordManipulation.cpp`.